CONTRIBUTING.md

```diff
@@ -0,0 +1,155 @@
+
+If you would like to provide a fix for a bug or a typo in the documentation or
+even add or change specific features of the role or specific parts of the
+documentation you are welcome to fork this repository, put the suggestion into
+a separate branch and create a pull request.
+Please also label the pull request, refer to related issues and assign
+reviewers that take care of reviewing your suggested changes.
+Be aware of the need to test your contributions before opening a pull request.
+Of course, before you do that we ask you to browse through the existing
+pull requests first to make sure that the bugs you would like to fix or
+enhancements you would like to propose are not covered by other pull requests
+already.
+
+### Do Testing
+
+Sometimes automated tests are not sufficient to account for specific corner 
+cases and to reveal hidden bugs.
+Testing the role manually is always a good approach to make sure that the role
+also works for other developers.
+Even more valuable are of course automated tests because they are repeatable.
+Please feel free to support the project by providing automated test cases.
+In order to do so you need to fork the repository, put the test cases into
+a separate branch and create a pull request.
+Please also use labels for your pull request, name issues that you refer to
+and assign reviewers who would be able to review your suggested test cases.
+Of course, before you do that we ask you to browse through the existing
+test cases first to make sure that the test cases you would like to propose
+are not covered by other test cases already.
+
+## Contributions Workflow via Pull Requests
+
+If you would like to contribute with pull requests for enhancements, bug-fixes
+or test cases, we kindly ask you to follow this workflow:
+
+![Development Workflow](docs/images/workflow.png)
+
+## Guidelines
+
+### Ansible Guides
+
+Because this is an Ansible project we agree with the 
+[Ansible Community Guide](https://docs.ansible.com/ansible/latest/community/index.html)
+and adhere to the
+[Ansible style guide](https://ansible.readthedocs.io/en/stable/dev_guide/style_guide/index.html).
+
+### Linting the Project
+
+Please make sure that your changes are linted before suggesting those changes
+in pull requests:
+
+```shell
+$ pipenv install --dev
+$ pipenv run molecule lint
+```
+
+### REUSE Specification Compliance
+
+Each file in this project needs to have a proper license and copyright header.
+Please check that each file you add to the project contains license and 
+copyright information and that it thereby meets the
+[REUSE Specification](https://reuse.software/spec/).
+
+```shell
+$ pipenv install --dev
+$ pipenv run reuse lint
+```
+
+### Testing the Ansible Role
+
+During development of your changes please use the existing automated test cases
+to verify that your changes do not break existing test cases:
+
+```shell
+$ pipenv install --dev
+$ pipenv run molecule test
+````
```

Comment:
To jump in. This is already done by CI (GitHub Actions).

Probably, this could be part of a section, how to setup a local development environment. It probably could be pushed more to the end of the file as it is not mandatory to have that.